compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/ConstantPropagation.java

```diff
@@ -274,6 +274,14 @@ public void visit(BLangInvocation bLangInvocationNode) {
         result = bLangInvocationNode;
     }
 
+    @Override
+    public void visit(BLangInvocation.BLangActionInvocation remoteMethodCallNode) {
```

Comment:
Shall we use 
```suggestion
    public void visit(BLangInvocation.BLangActionInvocation actionInvocation) {
```
instead?